app/assets/javascripts/questions.js

```diff
@@ -10,4 +10,16 @@ $(document).ready(function(q){
       $('.other-question').removeClass('hidden');
     });
 
+// Runs when the page loads and disables the submit button. Checks that both the answer and question text boxes are blank
+    if($('#question_text').val() || $('#question_answers_attributes_0_text').val() ==  "") 
+   $('#submit_question').attr('disabled', true);
```

Comment:
Space after `if`, one space between `==` and `""`, and use curly braces to wrap the second line as mentioned above.